src/scaict_uwu/service_wiring.py

```
@@ -0,0 +1,40 @@
+    }
+
+
+def get_config(service_container: ServiceContainer) -> Config:
+    return service_container.get_config_factory().get()
+
+
+def get_config_factory(service_container: ServiceContainer) -> ConfigFactory:
+    return ConfigFactory()
+
+
+def get_language_tag_factory(service_container: ServiceContainer) -> LanguageTagFactory:
```

Ruff (0.8.0)

Unused function argument: `service_container`

(ARG001)